Python/test-feh.py

```diff
@@ -0,0 +1,112 @@
+"""
+Test for DYNASIM FEH package.
+
+This script tests the DYNASIM feh_io module.
+"""
+
+from feh_io.read_feh import read_feh_data_file, read_parquet_2
```

Comment:
This is fine as is, but I just want to clarify what I meant when I ask you for a different kind of import. I had in mind something like this:
```
import feh_io
feh_io.read_feh_data_file
```
Or, with using `from`:
```
from feh_io import read_feh_data_file
```
I am not sure how to do that, but I think there is a way to specify names that a module exports in its `__init__.py`. This is definitely not a big deal, we can live with this.

Reply:
ok so this was accomplished with the prior configuration where the __init__.py combined all of the functions in read_feh and save_feh together. I don't think there's another way to do it unless you want the save module and the read module to be combined into one. Thoughts?

Reply:
I am not sure what "prior configuration" you're referring to. It seems that what I have in mind could be accomplished by import statements in `__init__.py`:
```
from .read_feh import read_feh_data_file
from .save_feh import save_feh_parquet
```
See item 2 in https://coderslegacy.com/python-init-py-best-practices/.

Reply:
ah sweet! I can implement this :) Thanks for pointing it out.

Reply:
addressed and tested!

Reply:
Thank you!